packages/cli/src/checks/__tests__/INTERNAL_MISMATCH.ts

```diff
@@ -85,4 +85,20 @@ describe("internal mismatch", () => {
       ]
     `);
   });
+
+  // regression test for https://github.com/Thinkmill/manypkg/issues/193
+  it.each(["npm:pkg-1@sometag", "npm:someorg@pkg-1@sometag"])(
```

Comment:
Shouldn't this be something like this?
```suggestion
  it.each(["npm:pkg-1@sometag", "npm:@someorg/pkg-1@sometag"])(
```